src/main/scala/temple/ast/OldServiceBlock.scala

```diff
@@ -0,0 +1,27 @@
+package temple.ast
+
+import temple.ast.Metadata.ServiceMetadata
+
+/** A service block, representing one microservice on its own isolated server */
+case class OldServiceBlock(
```

Comment:
I can't see where this is used? Also old vs what?

Reply:
Sorry, this should have been deleted following a merge conflict